frontend/management/src/components/Pagination.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import "../styles/pagination.css"
+import PaginationAll from "./PaginationAll.tsx";
+import PaginationLeft from "./PaginationLeft.tsx";
+import PaginationRight from "./PaginationRight.tsx";
+import PaginationCenter from "./PaginationCenter.tsx";
+
+function Pagination({ pageNumbers, itemsPerPage, lastPage, currentPage, handleClick } : Props) {
+  return (
+    <nav aria-label="Page navigation">
+      <ul className="pagination justify-content-end">
+        <li className={`page-item ${currentPage=== 1 ? "disabled" : ""}`}>
+          <a onClick={(e) => handleClick(e, currentPage-1)} className="page-link" href="#"> <i className="bi bi-caret-left"></i></a>
+        </li>
+        {
+          lastPage <= 7 || itemsPerPage == -1 ?
```

Comment:
Why 7?

Reply:
That is the number where the other design makes sense to use it.